thinc/tests/layers/test_softmax.py

```diff
@@ -55,7 +55,7 @@ def torch_softmax_with_temperature(
 
     return cast(
         Floats2d, torch2xp(torch.nn.functional.softmax(XWbt_temp, dim=-1))
-    ), cast(Floats2d, torch2xp(Xt.grad))
+    ), cast(Floats2d, torch2xp(cast(torch.Tensor, Xt.grad)))
```

Comment:
`mypy > 0.990` raises an error without `cast()`?

Reply:
Yes, otherwise it's `Optional[Tensor]`.